mozilla/fx-private-relay
# MPP-3577: String update for free onboarding

This PR fixes MPP-3577.

# New feature description
Some updates were made to strings for the free user onboarding experience, which were changed in pendingTranslations and the l10n repo.

Also, strings for the free onboarding experience were not added to the L10n repo previously, so a PR was created for that.
https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/163

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/59676643/33a84e34-f9b6-46ca-b027-e435fa9d8e73)

# How to test
* Turn on free_user_onboarding flag and verify in step 0 that the updatd "Lets create a new email mask" is shown.

# Checklist (Definition of Done)
